python/delete_instuck_progress.py

```diff
@@ -19,6 +19,8 @@ def drop_outdated_inprogress_file(abs_folder, filename):
     if os.path.isfile(os.path.join(abs_folder, filename)):
         mtime = os.stat(os.path.join(abs_folder, filename)).st_mtime
         now = time.gmtime()
```

Comment:
`datetime.utcnow()` looks nicer than `time.gmtime()`:

    >>> from datetime import datetime
    >>> import time
    >>> print(time.gmtime())
    time.struct_time(tm_year=2016, tm_mon=7, tm_mday=21, tm_hour=7, tm_min=53, tm_sec=53, tm_wday=3, tm_yday=203, tm_isdst=0)
    >>> print(datetime.utcnow())
    2016-07-21 07:55:49.649690

You can also use `datetime.utcfromtimestamp(mtime)` to convert the mtime to a datetime object.